<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> [Gpsd-dev] regressions
   </TITLE>
   <LINK REL="Index" HREF="http://lists.berlios.de/pipermail/gpsd-dev/2006-August/index.html" >
   <LINK REL="made" HREF="mailto:gpsd-dev%40lists.berlios.de?Subject=Re%3A%20%5BGpsd-dev%5D%20regressions&In-Reply-To=%3CPine.LNX.4.64.0608161710320.5040%40catbert.rellim.com%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="004132.html">
   <LINK REL="Next"  HREF="004122.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[Gpsd-dev] regressions</H1>
    <B>Gary E. Miller</B> 
    <A HREF="mailto:gpsd-dev%40lists.berlios.de?Subject=Re%3A%20%5BGpsd-dev%5D%20regressions&In-Reply-To=%3CPine.LNX.4.64.0608161710320.5040%40catbert.rellim.com%3E"
       TITLE="[Gpsd-dev] regressions">gem at rellim.com
       </A><BR>
    <I>Thu Aug 17 02:11:31 CEST 2006</I>
    <P><UL>
        <LI>Previous message: <A HREF="004132.html">[Gpsd-dev] SVN
</A></li>
        <LI>Next message: <A HREF="004122.html">[Gpsd-dev] regressions
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#4118">[ date ]</a>
              <a href="thread.html#4118">[ thread ]</a>
              <a href="subject.html#4118">[ subject ]</a>
              <a href="author.html#4118">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA1

Yo Eric!

I am trying to run the regressions and not having much luck.  I keep
getting things like this:

Processing test/garmin17n.log...Traceback (most recent call last):
  File &quot;/usr/bin/gpsfake&quot;, line 127, in ?
    test = gpsfake.TestSession(prefix=monitor, port=port, options=doptions, verbose=verbose)
  File &quot;/usr/local/lib/python2.3/gpsfake.py&quot;, line 347, in __init__
    self.daemon.wait_pid()
  File &quot;/usr/local/lib/python2.3/gpsfake.py&quot;, line 281, in wait_pid
    self.pid = int(fp.read())
ValueError: invalid literal for int():


Any ideas?  Python is not my thing.


RGDS
GARY
- ---------------------------------------------------------------------------
Gary E. Miller Rellim 20340 Empire Blvd, Suite E-3, Bend, OR 97701
	<A HREF="https://lists.berlios.de/mailman/listinfo/gpsd-dev">gem at rellim.com</A>  Tel:+1(541)382-8588
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v1.4.3 (GNU/Linux)

iD8DBQFE47Q28KZibdeR3qURAkddAKDTOoZ4Zr4dsqyI8uPOxqVIaoPh1QCdGlDj
O6OcrPTj7Wbwg0hMKqFvZnE=
=V6w+
-----END PGP SIGNATURE-----


</PRE>

<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="004132.html">[Gpsd-dev] SVN
</A></li>
	<LI>Next message: <A HREF="004122.html">[Gpsd-dev] regressions
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#4118">[ date ]</a>
              <a href="thread.html#4118">[ thread ]</a>
              <a href="subject.html#4118">[ subject ]</a>
              <a href="author.html#4118">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="https://lists.berlios.de/mailman/listinfo/gpsd-dev">More information about the Gpsd-dev
mailing list</a><br>
</body></html>
